server/lib/tuist/billing.ex

```diff
@@ -314,6 +325,46 @@ defmodule Tuist.Billing do
     }
   end
 
+  @doc """
+  Gets month-to-date LLM token usage for an account.
+  Returns a map: %{input: i, output: o, total: t} with zeros when no data.
+  """
+  def month_to_date_llm_token_usage(account_id) when is_integer(account_id) do
+    %Date{year: y, month: m} = Date.utc_today()
+    start_of_month_date = Date.new!(y, m, 1)
+    start_dt = DateTime.new!(start_of_month_date, ~T[00:00:00], "Etc/UTC")
```

Comment:
```suggestion
    start_dt = DateTime.new!(Date.utc_today() |> Date.beginning_of_month(), ~T[00:00:00], "Etc/UTC")
```

wouldn't this be slightly simpler since we have a `beginning_of_month` instead of using basically rewriting that ourselves?